exercises/pascals-triangle/introduction.md

```diff
@@ -0,0 +1,25 @@
+# Introduction
+
+With the weather being great, you're not looking forward to spending an hour in a classroom doing math.
+Annoyed, you enter the class room, where you notice a strangely satisfying triangle shape on the blackboard.
+Whilst waiting for your math teacher to arrive, you can't help but notice some patterns in the triangle: the outer values are all ones, each subsequent row has one more value than its previous row and the triangle is symmetrical.
+Weird!
+
+Not long after you sit down, your teacher enters the room and explains that this triangle is the famous
+[Pascal's triangle][wikipedia].
+
+Over the next hour, your teacher reveals some amazing things hidden in this triangle:
+
+- It can be used to compute how many ways you can pick K elements from N values.
+- It contains the Fibonacci sequence.
+- If you color odd and even numbers differently, you get a beautiful pattern called the [Sierpiński triangle][wikipedia-sierpinski-triangle]
+
+The teacher implores you and your classmates to lookup other uses, and assures you that there are lots more!
+At that moment, the school bell rings.
+You realize that for the past hour, you were completely absorbed in learning about Pascal's triangle.
+Math is definitely not as boring as you thought it was!
```

Comment:
As someone who doesn't think math is boring, I resent someone telling me that I thought that it was. 
How about something a bit more positive to build on the previous sentence?
 ```suggestion
Math can be really interesting after all!
```

Reply:
For the record: I also don't think math is boring (at all), I just couldn't come up with a better phrasing.

Reply:
I've just removed the section.